### PR TITLE
Added routes for rents

### PR DESCRIPTION
**Pull request by lucaspolonuer**

### CODE REVIEW

File: app/controllers/api/v1/rents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      def create
+        rent = Rent.create(user: current_user,
+                           book: Book.find(params[:book_id]),
+                           from: params[:from],
+                           to: params[:to])
+        render json: rent
+      end
+
+      def index
+        rents = Rent.where(user_id: current_user.id)
```
Comment (trueshizus) [inline app/controllers/api/v1/rents_controller.rb:13]: Avoid using _id unless is required

File: spec/controllers/api/v1/rents_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  end
+
+  describe 'POST #create' do
+    context 'When creating a rent' do
+      let!(:book) { create(:book) }
+      let!(:rent) { create(:rent, user: user, book: book) }
+
+      before { post :create, params: { id: user.id, book_id: book.id, from: rent.from, to: rent.to } }
+
+      it 'responses with the rent json' do
```
Comment (trueshizus) [inline spec/controllers/api/v1/rents_controller_spec.rb:33]: responds

File: Gemfile [hunk 3]
```diff
@@ -5,6 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+gem 'sidekiq'
```
Comment (trueshizus) [inline Gemfile:8]: Why is this gem added?

File: app/assets/javascripts/api/v1/rents.coffee [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (trueshizus) [inline app/assets/javascripts/api/v1/rents.coffee:1]: remove unused files

File: app/assets/stylesheets/api/v1/rents.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Api/V1/Rents controller here.
```
Comment (trueshizus) [inline app/assets/stylesheets/api/v1/rents.scss:1]: remove unused files

**Review by trueshizus — CHANGES_REQUESTED** [review]

**Comment by lucaspolonuer** [comment]
I will solve these in the next PR

**Pull request merged**